src/main/java/org/zalando/nakadi/service/converter/VersionZeroConverter.java

```diff
@@ -32,6 +43,71 @@
         return CursorConverter.Version.ZERO;
     }
 
+    public List<NakadiCursor> convertBatched(final List<SubscriptionCursorWithoutToken> cursors) throws
+            InvalidCursorException, InternalNakadiException, NoSuchEventTypeException, ServiceUnavailableException {
+        final NakadiCursor[] result = new NakadiCursor[cursors.size()];
```

Comment:
ok, maybe for this specific case when we map data by indexes it's ok to have arrays.